src/styles/_styles.scss

```diff
@@ -16,6 +16,7 @@
   padding: 8px;
   background: linear-gradient(180deg, rgba(0, 0, 0, 0.7) 0%, rgba(0, 0, 0, 0) 100%);
   width: 100%;
+  font-family: sans-serif;
```

Comment:
can we get it from the player? by `inherit` maybe

Reply:
no, `inherit` does't work here

Reply:
You don't need to specify it at all. The ui applies the fonts and the button will get the font from it.
Just remove the entire line

Reply:
done